Sources/Fluent/Preparation/Migration.swift

```diff
@@ -1,5 +1,5 @@
 final class Migration: Entity {
-    static var entity = "fluent"
+    static var entity = fluentTableName
```

Comment:
can we call this `migrationEntityName` 

- `fluent` is redundant especially considering this will be in `Config/fluent.json`
- NoSQL doesn't have a concept of "tables"